src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/in/NewStatus.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.model.in;
+
+public enum NewStatus {
+    CONSUMED,
+}
```

Comment:
Separate enums we only allow changing to some (well, one particular) statuses.
This will result in rejecting any requests with status that is not defined here.
Plus the swagger schema will be nicer (will only list valid statuses).